carbonmark/pages/api/bank-transfer/index.ts

```diff
@@ -0,0 +1,122 @@
+import { urls } from "lib/constants";
+import { NextApiRequest, NextApiResponse } from "next";
+
+type RequestBody = {
+  quantity: number;
+  email: string;
+  phone_number?: string;
+  company_name: string;
+  job_title?: string;
+  first_name: string;
+  last_name: string;
+  beneficiary_name?: string;
+  beneficiary_address?: string;
+  project_name: string;
+  retirement_message?: string;
+};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const {
+    quantity,
+    email,
+    phone_number,
+    company_name,
+    job_title,
+    first_name,
+    last_name,
+    beneficiary_name,
+    beneficiary_address,
+    project_name,
+    retirement_message = "",
+  } = req.body as RequestBody;
+
+  try {
+    const response = await fetch(urls.payViaBankForm, {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({
+        fields: [
+          {
+            name: "retirement_amount",
+            objectTypeId: "0-1",
+            value: quantity,
+          },
+          {
+            name: "email",
+            objectTypeId: "0-1",
+            value: email,
+          },
+          {
+            name: "phone",
+            objectTypeId: "0-1",
+            value: phone_number,
+          },
+          {
+            name: "company",
+            objectTypeId: "0-1",
+            value: company_name,
+          },
+          {
+            name: "jobtitle",
+            objectTypeId: "0-1",
+            value: job_title,
+          },
+          {
+            name: "firstname",
+            objectTypeId: "0-1",
+            value: first_name,
+          },
+          {
+            name: "lastname",
+            objectTypeId: "0-1",
+            value: last_name,
+          },
+          {
+            name: "who_will_this_retirement_be_credited_to_",
```

Comment:
Noob question: Why is there an underscore at the end of this?

Reply:
@biwano So this is the id that is automatically generated in hubspot based on the title - I think it adds an underscore for any special characters, spaces etc.. - in this case the title is "Who will this retirement be credited to?", so I assume it's adding the underscore at the end for the question mark.

<img width="1512" alt="Screenshot 2023-12-20 at 11 48 40" src="https://github.com/KlimaDAO/klimadao/assets/92357475/81d4e786-c782-4c05-b487-9b3d2355f93d">